libexec/apply_catalog.rb

```diff
@@ -58,7 +58,7 @@
   end
 
   # Ensure custom facts are available for provider suitability tests
-  Puppet::Node::Facts.indirection.find('puppetversion', environment: env)
+  facts = Puppet::Node::Facts.indirection.find('me', environment: env)
```

Comment:
Is there a more sensible name to put here? It doesn't appear to matter.

Reply:
are we concerned that the node name would find the wrong the facts? Should it be a random string?

Reply:
Probably should. There are also changes on the Puppet side, so this may need updates to work with Puppet 6.

Reply:
I don't think it'll find the wrong facts after BOLT-883.